rest_framework/fields.py

```diff
@@ -1183,6 +1186,9 @@ def to_internal_value(self, value):
         self.fail('invalid', format=humanized_format)
 
     def to_representation(self, value):
+        if not value:
```

Comment:
You shouldn't need to handle the `None` case explicitly, as I *believe* that's handled as a catch all.

What's the behavior if you *don't* include this?

Is this behavior in order to catch empty strings and return `None` for them? (In which case I suspect this may be valid)